iot-e2e-tests/common/src/main/java/com/microsoft/azure/sdk/iot/common/tests/iothubservices/twin/ReportedPropertiesTests.java

```diff
@@ -33,12 +35,14 @@ public ReportedPropertiesTests(String deviceId, String moduleId, IotHubClientPro
     }
 
     @Test
+    @ConditionalIgnoreRule.ConditionalIgnore(condition = StandardTierOnlyRule.class)
     public void testSendReportedProperties() throws IOException, IotHubException, InterruptedException
     {
         sendReportedPropertiesAndVerify(MAX_PROPERTIES_TO_TEST);
     }
 
     @Test
+    @ConditionalIgnoreRule.ConditionalIgnore(condition = StandardTierOnlyRule.class)
```

Comment:
>@ConditionalIgnoreRule.ConditionalIgnore [](start = 4, length = 40)

can this be applied at the class level instead?

Reply:
Currently can only apply to function, not class level

---
In reply to: [274580854](https://github.com/Azure/azure-iot-sdk-java/pull/485#discussion_r274580854) [](ancestors = 274580854)